test/input.txt

```diff
@@ -32,28 +35,42 @@
   add Valid Name p/12345 valid@email.butNoPrefix a/valid, address
   add Valid Name p/12345 e/valid@email.butNoAddressPrefix valid, address
   add Valid Name p/12345 e/valid@email.butNoTagPrefix a/valid, address t/goodTag noPrefixTag
+
+  # should still show that there are 0 entries
+  total
   
   # should catch invalid person data
   add []\[;] p/12345 e/valid@e.mail a/valid, address
   add Valid Name p/not_numbers e/valid@e.mail a/valid, address
   add Valid Name p/12345 e/notAnEmail a/valid, address
   add Valid Name p/12345 e/valid@e.mail a/valid, address t/invalid_-[.tag
 
-  # should add correctly and list non private information
+  # should still show that there are 0 entries
+  total
+
+  # should add correctly and list non private information and show correct total number of entries
   add Adam Brown p/111111 e/adam@gmail.com a/111, alpha street
   list
+  total
   add Betsy Choo pp/222222 pe/benchoo@nus.edu.sg pa/222, beta street t/secretive
   list
+  total
   add Charlie Dickson pp/333333 e/charlie.d@nus.edu.sg a/333, gamma street t/friends t/school
   list
+  total
   add Dickson Ee p/444444 pe/dickson@nus.edu.sg a/444, delta street t/friends
   list
+  total
   add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
   list
+  total
```

Comment:
Total function is tested too many times here. It is unnecessary and will only increase test time and margin for error.